config/crd/v1beta1/bus.volcano.sh_commands.yaml

```diff
@@ -1,6 +1,6 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
why do you change the version to V1 in the v1beta1 folder

Reply:
> why do you change the version to V1 in the v1beta1 folder, the base folder is v1 version

When install volcano with the script, it will be warning for `v1beta` is invalid.